sahha-android/src/main/java/sdk/sahha/android/data/manager/PermissionManagerImpl.kt

```diff
@@ -80,8 +80,32 @@ internal class PermissionManagerImpl @Inject constructor(
         return clientIsAvailable && isAndroid9OrAbove
     }
 
+    override suspend fun getTrimmedHcPermissions(
+        manifestPermissions: Set<String>?
+    ): Set<String> {
+        val permissions = getHcPermissions()
 
-    override suspend fun getHcPermissions(): Set<String> {
+        val trimmed = manifestPermissions?.let { mPermissions ->
```

Comment:
in an edgewise scenario where the set is empty, there would be no permissions

update to use the previous logic in this case